src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -249,4 +259,32 @@ private void scheduleEmailForSending(EmailWrapper email, long emailDelayTimer) {
         }
     }
 
+    /**
+     * Schedules for the search indexing of the student identified by {@code courseId} and {@code email}.
+     * @param startOfInterval time of start of creation
+     * @param intervalType the type of interval of the feedback response statistic
+     */
+    public void scheduleFeedbackResponseStatisticsCreation(Instant startOfInterval,
+            FeedbackResponseStatisticsType intervalType) {
+        Map<String, String> paramMap = new HashMap<>();
+
+        Instant endOfInterval;
+        if (intervalType == FeedbackResponseStatisticsType.HOUR) {
+            endOfInterval = startOfInterval.plusSeconds(Const.HOUR_IN_SECONDS);
+        } else {
+            endOfInterval = startOfInterval.plusSeconds(Const.MINUTE_IN_SECONDS);
+        }
+
+        paramMap.put(ParamsNames.FEEDBACK_RESPONSE_STATISTIC_STARTIME, Long.toString(startOfInterval.toEpochMilli()));
+        paramMap.put(ParamsNames.FEEDBACK_RESPONSE_STATISTIC_ENDTIME, Long.toString(endOfInterval.toEpochMilli()));
+        paramMap.put(ParamsNames.FEEDBACK_RESPONSE_STATISTIC_TYPE, intervalType.getValue());
+
+        addTask(TaskQueue.FEEDBACK_RESPONSE_STATISTICS_CREATION_QUEUE_NAME,
+                    TaskQueue.FEEDBACK_RESPONSE_STATISTICS_CREATION_WORKER_URL,
+                paramMap, null);
+        /*addCloudTask(TaskQueue.FEEDBACK_RESPONSE_STATISTICS_CREATION_QUEUE_NAME,
+                TaskQueue.FEEDBACK_RESPONSE_STATISTICS_CREATION_WORKER_URL,
+                paramMap, null);
+        */
```

Comment:
Not OK to have commented code block like this in production-ready codebase (dead code)